lightning-transaction-sync/src/esplora.rs

```diff
@@ -175,7 +180,8 @@ where
 				sync_state.pending_sync = false;
 			}
 		}
-		log_info!(self.logger, "Finished transaction sync.");
+		log_debug!(self.logger, "Finished transaction sync at tip {} in {}ms: {} confirmed, {} unconfirmed.",
```

Comment:
Hmm... I see. Could we at least modify the `log_error`s before returning to include the count up until then? Otherwise, it seems like we would be leaving out some counts. There also looks to be one case where we return an error but don't log.